skdist/tests/test_spark.py

```diff
@@ -0,0 +1,143 @@
+import pandas as pd
```

Comment:
Bringing up a point of discussion here. Should we have a separate test suite for spark functionality? Or mix in spark tests with the rest of the tests? I like having a separate `test_spark.py` script as done here. I'm curious if this will scale well as the library grows. Maybe this is the right way to start and we can course correct as needed going forward.

One downside here is that for local testing, a user will need to have local spark installed to run the test suite using a simple `pytest` command. I'm not sure if there is a clean way around this.

Reply:
We should discuss this more, as I think it impacts the future direction of the library. There is a fundamental we need to answer: "Will this library require Spark?" It probably _should_ given the nature of the functionality. If so, we should add clear setup instructions and tutorials for local Spark development. Thoughts?

Reply:
`Encoderizer` still provides useful functionality outside of a Spark environment, but is there anything else?

Reply:
Thats it. Though any of the estimators work without spark (`sc=None`), they wouldn't be terribly useful without it since they just mirror existing `sklearn` estimators. I think we should require `pyspark` and `pyarrow` with specific versions supported. It would be nice to keep tests that need spark and tests that don't in separate `test_*.py` scripts and somehow configure `pytest` to allow specifying which tests to run.